doc/classes/TileMapLayerGroup.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TileMapLayerGroup" inherits="Node2D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Groups a set of tile map layers together, allowing them to share a provided [TileSet].
+	</brief_description>
+	<description>
+	</description>
```

Comment:
Could use a description too, event explaining that the class is work in progress.
Though not sure how long it would be useful.

Reply:
I added a few lines for now, I guess it should be enough until we expose it properly.